myapp/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,43 @@
+class UsersController < ApplicationController
+  skip_before_action :logged_in_user, only: %i[new create]
+  before_action :editing_authority, only: %i[edit update]
+
+  def new
+    redirect_to root_path and return unless current_user.nil?
```

Comment:
```suggestion
    redirect_to root_path and return if current_user.present?
```
の方がわかりやすそうですね

ref: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/UnlessElse

Reply:
ご指摘ありがとうございます！
こちら修正しました。

https://github.com/Fablic/training/pull/1175/commits/ae4460e6d0e8d76a0318be49c16e7bbf62050e41